templates/server/puppetserver/conf.d/auth.conf.erb

```diff
@@ -56,6 +56,16 @@ authorization: {
             name: "puppetlabs environments"
         },
         {
+            match-request: {
+                path: "/puppet/v3/resource_type"
```

Comment:
I've just created https://github.com/theforeman/puppet-puppet/pull/388 which should probably be applied before this PR (which may then need a rebase).